pkg/scheduler/core/generic_scheduler.go

```diff
@@ -19,7 +19,7 @@ import (
 
 // ScheduleAlgorithm is the interface that should be implemented to schedule a resource to the target clusters.
 type ScheduleAlgorithm interface {
-	Schedule(context.Context, *policyv1alpha1.Placement, *workv1alpha2.ResourceBindingSpec, *ScheduleAlgorithmOption) (scheduleResult ScheduleResult, err error)
```

Comment:
I'm not sure if we need to add this parameter back when implementing #3126, but it makes sense for this PR to do it.